api/ring-camera.ts

```diff
@@ -59,6 +73,7 @@ export class RingCamera {
       this.batteryLevel !== null &&
       this.batteryLevel < 100 &&
       this.batteryLevel >= 0)
+  existingDoorbellType = getExistingDoorbellType(this.initialData)
```

Comment:
It would probably be best to change this to an observable so that we can subscribe to it in the homebridge layer and to prevent stale data being accessible via the api

```suggestion
  onExistingDoorbellType = this.onData.pipe(
    map(getExistingDoorbellType),
    distinctUntilChanged()
  )
```

Reply:
Makes sense! I implemented this incorrectly in https://github.com/dgreif/ring/pull/152/commits/a7fa4a2eeade1c2d0ef4454a0927b74371d126fd — after reading through the `lock.ts` code I see what I ought to have done. Working to update this PR now.

Reply:
Done—fixed in https://github.com/dgreif/ring/pull/152/commits/e13c4bb1a1ecff5f9cb71f539ca90348c241764c with a minor additional fix in https://github.com/dgreif/ring/pull/152/commits/08ce82871015b3df669703572bda9f253d33fbd1